components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -440,14 +442,20 @@
 <config_use_activeTracers>.true.</config_use_activeTracers>
 <config_use_activeTracers_surface_bulk_forcing>.true.</config_use_activeTracers_surface_bulk_forcing>
 <config_use_activeTracers_surface_restoring>.false.</config_use_activeTracers_surface_restoring>
+<config_use_activeTracers_surface_restoring ocn_grid="oARRM60to10">.true.</config_use_activeTracers_surface_restoring>
+<config_use_activeTracers_surface_restoring ocn_grid="oARRM60to6">.true.</config_use_activeTracers_surface_restoring>
```

Comment:
these should not be default because we only do restoring in G-cases.

Reply:
I will back out all of the changes to the mpaso namelist_defaults.  I think that is more consistent with the overall strategy of using user_nl_mpaso to set things up and the benefit is the namelist defaults file remains consistent with master.